File: Beecrowd/1329 Cara ou Coroa/main.py
while True:
    n = int(input())

    if n == 0:
        break

    cont0 = 0
    cont1 = 0

    l  = [int(_) for _ in input().split()[:n]]

    for i in range(n):
        if l[i] == 0:
            cont0 += 1
        elif l[i] == 1:
            cont1 += 1

    print(f"Mary won {cont0} times and John won {cont1} times")

File: Beecrowd/1329 Cara ou Coroa/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int N, L, i, cont0, cont1;

    do
    {
        cin >> N;

        if (N != 0)
        {
            cont0 = 0;
            cont1 = 0;

            for (i = 0; i < N; i++)
            {
                cin >> L;

                if (L == 0)
                    cont0++;
                if (L == 1)
                    cont1++;
            }
            cout << "Mary won " << cont0 << " times and John won " << cont1 << " times" << endl;
        }
    } while (N != 0);

    return 0;
}